src/klicky.ts

```diff
@@ -1,3 +1,21 @@
-export const Klicky = () => {
+export const Klicky = (selector: string) => {
```

Comment:
Can you update this PR with the `data-*` selector work we talked about?

Reply:
PR has been updated.